.php_cs

```diff
@@ -23,8 +23,13 @@ return PhpCsFixer\Config::create()
         '@PHP71Migration' => true,
         '@PHP71Migration:risky' => true,
         '@PHP73Migration' => true,
+        '@PHP74Migration' => true,
```

Comment:
All this CS maintenance things we should switch to `doctrine/coding-standard`, so much simpler than maintaining this ruleset I think .... thoughts?

Reply:
All 3 projects (`browscap`, `browscap-php` and `browscap-site`) have their own coding styles and tooling sets. If we change the coding standard, we should unify the coding standard and the tools over all projects.

Reply:
I am ok with that; `doctrine/coding-standard` is easy to pull in & apply (`vendor/bin/phpcs` to check, `vendor/bin/phpcbf` to update).